website/src/components/LeaderboardGridCell/LeaderboardGridCell.tsx

```diff
@@ -0,0 +1,97 @@
+import { Avatar, Box, Grid, GridItem, Text, useColorModeValue } from "@chakra-ui/react";
+import { FiChevronDown } from "react-icons/fi";
+import { get } from "src/lib/api";
+import useSWR from "swr";
+
+/**
+ * Presents a grid of leaderboard entries with more detailed information.
+ */
+const LeaderboardGridCell = () => {
+  const { data: leaderboardEntries } = useSWR("/api/leaderboard", get);
+  const backgroundColor = useColorModeValue("white", "gray.800");
+  const columns = `repeat(${FILTER.length}, 1fr)`;
+
+  return (
+    <>
+      <Grid>
+        <GridItem
+          colSpan={4}
+          bg={backgroundColor}
+          display="grid"
+          gridTemplateColumns={columns}
+          p="4"
+          borderRadius="lg"
+          mb="4"
+          shadow="base"
+        >
+          {FILTER.map(({ title, GridItemProps }, index) => (
+            <GridItem key={index} display="flex" {...GridItemProps}>
+              <Box display="flex" alignItems="center" gap="2" width="fit-content" borderRadius="md" cursor="pointer">
+                <Text fontSize="sm" fontWeight="bold" textTransform="uppercase">
+                  {title}
+                </Text>
+
+                <FiChevronDown size="16" />
+              </Box>
+            </GridItem>
+          ))}
+        </GridItem>
+      </Grid>
+      <Grid templateColumns={columns} bg={backgroundColor} borderRadius="xl" shadow="base" p="4" gap="6">
```

Comment:
is there a  reason why these need to be 2 separate grids?

the columns won't be aligned between the headers & the rows.

Reply:
Right now they line up well enough with the column fields. 

But I think the answer is no, they can be the same grid or a simple table.  Ultimately I think this component and `LeaderboardTable` should be the same.  But I want to assign that as a UI cleanup to Riley as he's better at that than I am.  I mostly wanted to do a little re-organizational cleanup and fold in the real data.